.github/workflows/update-snapshots.yaml

```diff
@@ -0,0 +1,50 @@
+name: Update Snapshots
+ 
+on: workflow_dispatch
+
+permissions:
+  contents: write 
+  pull-requests: write 
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+    - name: Setup yarn
+      run: |
+        cd ./integration-tests/chopsticks
```

Comment:
Please add a comment that this is copy-pasted from our other CI step, so we can link the two together.

Reply:
Comment added in: [b57b55d](https://github.com/KILTprotocol/kilt-node/pull/636/commits/b57b55dd1b29ae84b7ad429de93691f4e867ea7a)